source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -0,0 +1,68 @@
+---
+title: Writing Python at GDS
+---
+
+# <%= current_page.data.title %>
+
+
+### About this manual:
+
+This manual is designed to aid developers in writing Python code that is clear and consistent, within, and across,
+projects at GDS.
+
+We follow [PEP8][], in some cases [PEP8][] doesn't express a view (e.g. on the usage of language
+features such as metaclasses) in these cases we defer to the [Google Python style guide][GPSG].
+We suggest using [PEP8][] and the [Google Python style guide][GPSG] (in order of preference) as references unless
+something is explicitly mentioned in the [GDS Python Style Guide][GDSPSG].
+
+
+### Python 3
+
+Prefer Python 3. Python 2 will [reach end of life in 2020][PEP373].
+Python 3 is now well-supported by application frameworks and libraries, and is commonly used in production.
+
+If you've never migrated a project from Python2 to Python3 there are plenty of resources online and it is usually
+surprisingly easy.
+
+
+### PEP8
+
+The only allowable exceptions to [PEP8][] should be detailed in the [GDS Python Style Guide][GDSPSG] or in the
```

Comment:
I don't think my comment at https://github.com/alphagov/gds-tech/pull/71#discussion_r142084542 has been properly addressed?  This still contradicts the "know when to be consistent" statement in the generic style manual.

Reply:
Updated